src/__tests__/animeSlice.test.js

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+import configureMockStore from 'redux-mock-store';
```

Comment:
- [ ] Great job so far on building your capstone project. However, While reviewing your test, I noticed one of your tests is failing due to a `module` that can't be found. Believe is full details of the failing test details. Kindly take a look at it and please make sure to fix it.
<img width="677" alt="Screenshot 2023-08-07 at 22 09 43" src="https://github.com/kit0-0/react-metrics-anime/assets/107192076/105fac04-5644-4c8a-aab4-2aa582408b76">